compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -453,7 +454,10 @@ private Manifest createManifest() {
     private void copyJar(ZipArchiveOutputStream outStream, JarLibrary jarLibrary,
             HashMap<String, JarLibrary> copiedEntries, HashMap<String, StringBuilder> services) throws IOException {
 
-        ZipFile zipFile = new ZipFile(jarLibrary.path().toFile());
+        String pkgName = jarLibrary.packageName().get();
+        HashMap<String, HashSet<String>> classWiseDeadFunctionMap = getDeadFunctionList(pkgName, this.compilerContext);
```

Comment:
```suggestion
        HashMap<String, HashSet<String>> classWiseDeadFunctionMap = getDeadFunctionList(jarLibrary.packageName().get(), this.compilerContext);
```

Is there any usages of pkgName within this method?

Reply:
Changed in commit 70012d7a0af8c168e0ffe63d303385078e4caa8d